Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -805,51 +685,69 @@ void ExecuteCommand(u32 command_0, u32 command_1, u32 command_2, u32 output_addr
     break;
 
   // Seems to be used by both GC and Wii
-  case DVDLowSeek:
+  case SEEK:
     // Currently unimplemented
     INFO_LOG(DVDINTERFACE, "Seek: offset=%09" PRIx64 " (ignoring)", (u64)command_1 << 2);
     break;
 
   // Probably only used by Wii
-  case DVDLowReadDvd:
+  case READ_DVD_METADATA:
+    switch ((command_0 >> 16) & 0xFF)
+    {
+    case 0:
+      ERROR_LOG(DVDINTERFACE, "DVDLowReadDvdPhysical");
+      break;
+    case 1:
+      ERROR_LOG(DVDINTERFACE, "DVDLowReadDvdCopyright");
+      break;
+    case 2:
+      ERROR_LOG(DVDINTERFACE, "DVDLowReadDvdDiscKey");
+      break;
+    default:
+      ERROR_LOG(DVDINTERFACE, "Unknown 0xAD subcommand in %08x", command_0);
+      break;
+    }
+    break;
+  // Probably only used by Wii
+  case READ_DVD:
     ERROR_LOG(DVDINTERFACE, "DVDLowReadDvd");
     break;
   // Probably only used by Wii
```

Comment:
You can remove these comments about GC vs Wii and direct access vs IOS access. I added them as a compromise for the code not showing which ones only are usable via direct access or IOS on a real console, but now the code does show that. (See PR #1634 for reference.)

Reply:
Sure, I can do that.  At this point I'm more or less completely sure that the list on gc-forever/[the old dvd-game wiki](https://web.archive.org/web/20070708050423/http://tmb.elitedvb.net/dvd-game/index.php/DICommands) is correct, with the one exception being the debug commands (with [0xFF](https://web.archive.org/web/20070328200145/http://tmb.elitedvb.net/dvd-game/index.php/CMDFF) at least not seeming to work as-is on the Wii, but [the same wiki](https://web.archive.org/web/20070328200323/http://tmb.elitedvb.net/dvd-game/index.php/CMDFE) claims that the Wii introduced new 0xFE debug commands, so I don't know what to do with that.)  (There is one other thing that I haven't yet tested: which commands can be used with dvd-video disabled.  But those aren't emulated yet, so it doesn't really matter.)